libpod/healthcheck.go

```diff
@@ -428,7 +472,48 @@ func (c *Container) getHealthCheckLog() (define.HealthCheckResults, error) {
 		return healthCheck, fmt.Errorf("failed to read health check log file: %w", err)
 	}
 	if err := json.Unmarshal(b, &healthCheck); err != nil {
-		return healthCheck, fmt.Errorf("failed to unmarshal existing healthcheck results in %s: %w", c.healthCheckLogPath(), err)
+		return healthCheck, fmt.Errorf("failed to unmarshal existing healthcheck results in %s: %w", path, err)
+	}
+	return healthCheck, nil
+}
+
+func (c *Container) readFromEventLoggerHealthCheckLog() (define.HealthCheckResults, error) {
+	var healthCheck define.HealthCheckResults
+	filters := []string{
+		fmt.Sprintf("container=%s", c.ID()),
+		fmt.Sprintf("event=%s", events.HealthLog.String()),
+		"type=container",
+	}
+
+	containerEvents, err := c.runtime.GetEvents(context.Background(), filters)
```

Comment:
> So, from reading this - do we support logging to file and journald _at the same time_? That seems unnecessary - one or the other should be sufficient.

I did it according to @mheon's comment. But yes. I will extend the `healt_status` event with a log and use a file so that it doesn't affect performance as I wrote it in the design doc.